pkg/controller/migplan/registry.go

```diff
@@ -83,44 +82,37 @@ func (r ReconcileMigPlan) ensureMigRegistries(plan *migapi.MigPlan) error {
 }
 
 // Ensure the credentials secret for the migration registry on the specified cluster has been created
-func (r ReconcileMigPlan) ensureRegistrySecret(client k8sclient.Client, plan *migapi.MigPlan, storage *migapi.MigStorage) error {
+func (r ReconcileMigPlan) ensureRegistrySecret(client k8sclient.Client, plan *migapi.MigPlan, storage *migapi.MigStorage) (string, error) {
```

Comment:
Do you think there's any value in returning the secret found/created here rather than just a string to it? Wondering if we would need to access it from anywhere else in the reconcile.

Reply:
> Was thinking that returning an actual `secret` type here might make it clearer what you get back from this method.

Thinking this too.

Reply:
Sure. I can return the whole thing and then just call GetName() as needed down the line.

Reply:
+1 this makes sense to me.